index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <main id="main">
```

Comment:
Please use a semantic tag, e.g section to ensure accessibility and SEO ✨⛏ and reader readability as well you can add a section tag and put all the content of the main into this particular section tag